packages/perseus/src/types.ts

```diff
@@ -100,7 +100,6 @@ export interface Widget {
     getUserInputMap?: () => UserInputMap | undefined;
     getUserInput?: () => UserInputArray | UserInput | undefined;
 
-    showRationalesForCurrentlySelectedChoices?: (options?: any) => void;
```

Comment:
There's a `deselectIncorrectSelectedChoices` in this type also (line 77). Does that need to go away also?

Reply:
Yes! Thank you, I think I've cleared them all out now.